website/server/api/routers/categoryRouter.js

```diff
@@ -1,10 +1,10 @@
 import express from 'express';
-
+import passport from 'passport';
 import categoryController from '../controllers/categoryController';
 
 let categoryRouter = express.Router();
 
 let categoryControllerInstance = new categoryController();
-categoryRouter.get('/', categoryControllerInstance.listAllCategories);
+categoryRouter.get('/', passport.authenticate('jwt', {session: false}),categoryControllerInstance.listAllCategories);
```

Comment:
Let's create a middleware function, don't need to copy and paste this line of code all around the code-base.